### PR TITLE
New component styles

### PR DESCRIPTION
**Pull request by afc163**
New UI !!! 🎉 

- [x] Default font size => 14px
- [x] Bigger components size
- [x] Don't set large size of form controls defaultly
- [x] New shadows
- [x] New color systems
- [x] Tweak default padding of controls and items
- [x] compatibility reset styles for 2.x
- [x] More new icons

---

Tweak and check all other components:

- [x] affix
- [x] alert
- [x] anchor
- [x] auto-complete
- [x] avatar
- [x] back-top
- [x] badge
- [x] breadcrumb
- [x] button
- [x] calendar
- [x] card
- [x] carousel
- [x] cascader
- [x] checkbox
- [x] collapse
- [x] date-picker
- [x] divider
- [x] dropdown
- [x] form
- [x] grid
- [x] icon
- [x] input
- [x] input-number
- [x] layout
- [x] list
- [x] mention
- [x] menu
- [x] message
- [x] modal
- [x] notification
- [x] pagination
- [x] popconfirm
- [x] popover
- [x] progress
- [x] radio
- [x] rate
- [x] select
- [x] slider
- [x] spin
- [x] steps
- [x] switch
- [x] table
- [x] tabs
- [x] tag
- [x] time-picker
- [x] timeline
- [x] tooltip
- [x] transfer
- [x] tree
- [x] tree-select
- [x] upload

### CODE REVIEW

**Comment by afc163** [comment]
```
投影层级自下而上：

第一层： 卡片 hover 反馈
#000000（ 9%）  x：0    y：2   blur：8

第二层： 工作区内组件呼出反馈（如dropdown、select、popover、tooltips、data picker 等）
#000000（ 15%）  x：0    y：2   blur：8

第三层：工作区内全局反馈（如 message、notification 等）
#000000（ 15%）  x：0    y：4   blur：12

导航区层：设计师自灵活自定

最高层：modal
遮罩：#000000（65%）
阴影：#000000（ 45%）  x：0    y：4   blur：12
```

For note

File: components/style/v2-compatible-reset.less [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+// unify the setting of elements's margin and padding for browsers
+body,div,dl,dt,dd,ul,ol,li,h1,h2,h3,h4,h5,h6,pre,code,form,fieldset,legend,input,textarea,p,blockquote,th,td,hr,button,article,aside,details,figcaption,figure,footer,header,hgroup,menu,nav,section {
+  margin: 0;
+  padding: 0;
+}
+
+ul,
+ol {
+  list-style: none;
+}
```
Comment (afc163) [inline components/style/v2-compatible-reset.less]: https://github.com/ant-design/ant-design/issues/4331#issuecomment-331328304

**Comment by afc163** [comment]
All done!!! :tada:

**Comment by LinBoLen** [comment]
feels like there is a budge of work for me to sync style into scss. which have not been fully tested yet

**Pull request merged**